eng/pipelines/patch-release.yml

```diff
@@ -3,7 +3,8 @@ pr: none
 extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client-patch.yml
   parameters:
-    artifacts:
+    TestPipeline: true
```

Comment:
What does the `TestPipeline` property do?

Reply:
@vcolin7 I know we'd chatted over teams but I'm putting an answer here so everyone knows.
This change is temporary and will be removed prior to submission. TestPipeline is necessary for releasing the set of template libraries. [eng/common/pipelines/templates/steps/set-test-pipeline-version.yml](https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/pipelines/templates/steps/set-test-pipeline-version.yml) needs this to be set to manipulate the versions/update changelog entries so we release the template libraries without requiring version prep PRs.